ST_grid.c

```diff
@@ -631,9 +634,9 @@ static void _run_spinup(void)
 			for (spinup_Cell = 0; spinup_Cell < nSoilTypes; spinup_Cell++)
 			{ // for each different soil type
 
-				int cell = soilTypes_Array[spinup_Cell]; // this is the first cell of this soiltypes actual cell number
-				int j = cell % grid_Cols + 1; // this is the column of the first cell of this soiltype
-				int i = ((cell + 1 - j) / grid_Cols) + 1; // this is the row of the first cell of this soiltype
+				//int cell = soilTypes_Array[spinup_Cell]; // this is the first cell of this soiltypes actual cell number
```

Comment:
There is a lot of commented out code in ST_grid.c. If we plan on overhauling grided mode eventually we can hold off on fixing this.

Reply:
Let's hold off for now, but document this as an issue in the milestone gridded_code